website/scripts/build-static-content.js

```diff
@@ -459,7 +459,7 @@ module.exports = {
                   throw new Error(`Failed compiling markdown content: An article page is missing a category meta tag (<meta name="category" value="guides">) at "${path.join(topLvlRepoPath, pageSourcePath)}".  To resolve, add a meta tag with the category of the article`);
                 } else {
                   // Throwing an error if the article has an invalid category.
-                  let validArticleCategories = ['deploy', 'security', 'engineering', 'success stories', 'announcements', 'guides', 'releases', 'podcasts', 'report' ];
+                  let validArticleCategories = ['deploy', 'security', 'engineering', 'success stories', 'announcements', 'guides', 'releases', 'podcasts', 'report', 'case studies' ];
```

Comment:
The name for case studies currently is "success stories".  If we want to change that, we can set up redirects in routes, but if we change it, then we should remove "success stories", not keep both.

Reply:
Agreed

Reply:
@mikermcneil, @jarodreyes, I considered our current "success stories" (or "user stories" as framed in the articles) as different post types. They're shorter format and more casual in style. Whereas case studies are longer, challenge/solution-type articles. Happy to remove success stories if you guys prefer, though.

Reply:
@eashaw, I confirmed with Jarod that success stories and case studies should be categorized as "case studies." So please remove "success stories" and set up redirects.

Reply:
@eashaw, but let's wait until we hear back with approval to publish before spending time on this.